src/airtop/wrapper/windows_client.py

```diff
@@ -75,7 +79,7 @@ def prompt_content(
             request_options = RequestOptions(timeout_in_seconds=600)
         elif request_options.get("timeout_in_seconds") is None:
             request_options.update({"timeout_in_seconds": 600})
-        return super().prompt_content(session_id, window_id, prompt=prompt, client_request_id=client_request_id, cost_threshold_credits=cost_threshold_credits, follow_pagination_links=follow_pagination_links, time_threshold_seconds=time_threshold_seconds, request_options=request_options)
+        return super().prompt_content(session_id, window_id, prompt=prompt, client_request_id=client_request_id, configuration=configuration, cost_threshold_credits=cost_threshold_credits, follow_pagination_links=follow_pagination_links, time_threshold_seconds=time_threshold_seconds, request_options=request_options)
```

Comment:
Needed to sync the windows_client as the base client now has a different signature for both prompt and scrape, and now receives a configuration object.